packages/internals/tsconfig.build.json

```diff
@@ -3,8 +3,5 @@
   "compilerOptions": {
     "outDir": "dist"
   },
-  "include": ["src"],
-  // to not copy local engines to dist
-  // so npm doesn't publish them
-  "exclude": ["src/get-generators/engines"]
```

Comment:
I believe having the equivalent in the git ignore was enough, this is really just for type generation.